lib/ansible/modules/windows/win_domain_computer.ps1

```diff
@@ -0,0 +1,183 @@
+#!powershell
+#
+# Copyright: (c) 2017, AMTEGA - Xunta de Galicia
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#Requires -Module Ansible.ModuleUtils.Legacy
+
+
+# ------------------------------------------------------------------------------
+$ErrorActionPreference = "Stop"
+
+# Preparing result
+$result = @{}
+Set-Attr $result "changed" $false
+
+# Parameter ingestion
+$params = Parse-Args $args -supports_check_mode $true
+
+$check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -default $false | ConvertTo-Bool
```

Comment:
Instead of using the `ConverTo-Bool` option you can set `-type "bool"` on the `Get-AnsibleParam` and it will convert it for you, e.g. it would look like

`Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false`